edx_proctoring/templates/proctoring/proctored-exam-status.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="exam-timer">
+<%- gettext("You are taking") %>
+<a href="<%- interpolate(gettext('%(exam_url_path)s'), { exam_url_path: exam_url_path }, true)%>" >
+
+<%- interpolate(gettext(' %(exam_display_name)s '), { exam_display_name: exam_display_name }, true) %>
+</a>
+<%- gettext(" exam as a proctored exam. Good Luck!") %>
```

Comment:
Still working on some wording that could provide info about the timed aspect/expiring time warning where we currently have "Good Luck!"